front/app/modules/commercial/insights/services/insightsCategoriesSuggestionsTasks.ts

```diff
@@ -38,7 +38,6 @@ export function insightsCategoriesSuggestionsTasksStream(
       insightsViewId
     )}`,
     ...streamParams,
-    cacheStream: false,
```

Comment:
Not sure this is ok, we probably actually want to refetch the tasks each time we query for them since they're so likely to change

Reply:
Indeed, not sure why this was changed! Thanks for spotting :)